src/test/java/org/llorllale/cactoos/matchers/MismatchesTest.java

```diff
@@ -27,15 +27,20 @@
 
 package org.llorllale.cactoos.matchers;
 
+import org.cactoos.Text;
 import org.cactoos.text.TextOf;
+import org.hamcrest.Description;
+import org.hamcrest.StringDescription;
+import org.hamcrest.core.IsEqual;
 import org.junit.jupiter.api.Test;
 
 /**
  * Test case for {@link Mismatches}.
  *
  * @since 1.0.0
+ * @checkstyle JavadocMethodCheck (500 lines)
  */
-final class MismatchesTest {
+public final class MismatchesTest {
```

Comment:
@baudoliver7 no need to add this, all our test class are non public

Reply:
@victornoel You're totally right !